app.js

```diff
@@ -1,37 +1,22 @@
-
-/**
- * Module dependencies.
- */
-
-var express = require('express')
-  , routes = require('./routes')
-  , fs = require('fs')
-  , settings = require('./settings')
-  , port = 8080
-  , http = require('http');
-
+var express = require('express');
```

Comment:
Worthwhile noting here that you could have kept my comma declarations for the variables at the top of the file.